src/pages/Faq/components/FaqItem.scss

```diff
@@ -0,0 +1,3 @@
+.text-box{
```

Comment:
do you know that this style will apply to all elements with "text-box" class in your applivation?)